gui/velociraptor/src/themes/veloci-docs.css

```diff
@@ -1,20 +1,45 @@
+@font-face {
+  font-family: "Iosevka Term";
+  font-style: normal;
+  font-weight: 500;
+  src: local(""),
+    url("../fonts/iosevka-term-medium-subset.woff2") format("woff2");
+  unicode-range: U+0020-007E, U+0080-00FF;
+  font-display: block;
+}
+
 .veloci-docs .modal-backdrop {
-    /* Reduce opacity for better illustration */
-    --bs-backdrop-opacity: .15;
+  /* Reduce opacity for better illustration */
+  --bs-backdrop-opacity: 0.15;
 }
 
 body.veloci-light.veloci-docs {
   font-weight: 500;
 }
 
-
 .veloci-light.veloci-docs {
-    --color-foreground: #00000;
-    --color-foreground-inverse: #00000;
+  --color-foreground: #000000;
```

Comment:
This is actually wrong - it makes the foreground transparent (alpha is 0) which break a lot of things.

Reply:
No, yours had a zero missing. It's RGB so either `#000000` or `#000`. Transparent would be 8 zeros.

Reply:
RGBA would be 4 or 8 digits